lib/util/date.ts

```diff
@@ -1,4 +1,6 @@
-import { formatDistance } from 'date-fns';
+// TODO: Remvove the deep imports once we start offering an esm build
+// and confirm that tree-shaking date-fns still works.
```

Comment:
Actually, I think that the CLI would benefit from a deep import, since it will always have to parse less files, regardless of switching to esm or not :thinking: